UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -421,12 +427,30 @@ public class Background : UndertaleObject, INotifyPropertyChanged
             private UndertaleRoom _ParentRoom;
 
             /// <summary>
-            /// The room parent this background belongs to.
+            /// The room parent this background belongs to.<br/>
+            /// Set by <see cref="SetupRoom(bool)"/>.
             /// </summary>
+            /// <remarks>
+            /// This attribute is UMT-only and does not exist in GameMaker.
+            /// </remarks>
             public UndertaleRoom ParentRoom { get => _ParentRoom; set { _ParentRoom = value; OnPropertyChanged(); UpdateStretch(); } }
 
-            //TODO:
+            /// <summary>
+            /// The calculated horizontal render scale for the background texture.<br/>
+            /// Used in the UndertaleModTool room editor.
```

Comment:
This sentence is unnecessary. An UndertaleRoom attribute being used by the room editor is obvious.